plugins/auth/index.js

```diff
@@ -113,15 +113,30 @@ exports.register = (server, options, next) => {
                 accessTokenName: 'access_key',
                 allowCookieToken: false,
                 allowQueryToken: true,
-                validateFunc: (token, cb) => {
-                    if (token !== pluginOptions.temporaryAccessKey) {
-                        return cb(null, false, {});
+                validateFunc: function _validateFunc(token, cb) {
```

Comment:
Apparently the `server` object doesn't actually have `server.app.userFactory`, to get that you need to get `server._parent.app.userFactory`. Using a private property seems suspect, and ESLint actually won't allow it. 

Between using `server._parent` and disabling the ESLint rule for the line or using `function` notation, I feel like `function` is cleaner.